### ISSUE TITLE
UI5 Carousel - Add property to allow content alignment adjustments

### ISSUE THREAD

**Issue by veredcon** [SAP/ui5-webcomponents#5697]
### **Feature Request Description**

Regarding UI5 Carousel - https://sap.github.io/ui5-webcomponents/playground/components/Carousel/
Add property/attributes that will allow to control the alignment of the items content.

For example in this case, we would like to align the items to the left as the title of the carousel is aligned left.
![image](https://user-images.githubusercontent.com/9718939/185881228-fb5294c3-25f2-4381-bbe3-9db12e94ec39.png)

Today the default is center and cannot be configured.

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

### **Stakeholder Info**
BAS

- Business impact: Layout of the Welcome Page of the Business Application Studio based VSCode

**Comment by TeodorTaushanov** [comment]
Hi @veredcon,

Now you can style the carousel content part. In your case you can add a class to the ui5-carousel:
`<ui5-carousel class="carouselContentLeftAligned"`

and you can left align the content items with:

```
.carouselContentLeftAligned::part(content) {
	align-items: flex-start;
}
```

Best,
Teo

**Comment by veredcon** [comment]
Hi @TeodorTaushanov

Thanks for the quick addition!
In which @ui5/webcomponents version will it be included?

Thanks!
Vered

**Comment by TeodorTaushanov** [comment]
Hi @veredcon,

This functionality will be available from version 1.7. You can check the roadmap here - https://github.com/SAP/ui5-webcomponents/projects?type=classic

Best,
Teo
